src/coreservices.cpp

```diff
@@ -304,6 +304,7 @@ void CoreServices::initialize(QApplication* pApp) {
     }
 
     m_pPlayerManager->addConfiguredDecks();
+    // addConfiguredSamplers() ?
```

Comment:
This should be removed

Reply:
:+1: slipped in, and has already been added